dashboard/app/controllers/projects_controller.rb

```diff
@@ -696,4 +753,49 @@ def extra_links
       view_options(responsive_content: true)
     end
   end
+
+  # Temporary - will be replaced with storing in database.
+  private def send_project_submission(name, username, project_type, channel_id, description)
+    unless Rails.env.development? || Rails.env.test?
+      subject = 'TESTING: Featured project gallery submission'
+      response = HTTParty.post(
+        'https://codeorg.zendesk.com/api/v2/tickets.json',
```

Comment:
Optional: we have other places in the codebase where we send tickets to zendesk using the zendesk API right? Maybe worth consolidating into a ZendeskHelper or similar if there's a lot of repeated code?

Reply:
Yes - this currently used in `report_abuse_controller`. I thought about this, but this is temporary code which will be removed once we store submissions in a table. We do not want to use ZenDesk to send featured project submissions in the long-term. Thanks!

Reply:
Agree, seems reasonable to me to just duplicate in this case :)